### PR TITLE
fixes for md

### LINKED ISSUE #485: NER: output of lbjava
@cowchipkid have you checked in these files intentionally? 
https://github.com/CogComp/cogcomp-nlp/tree/master/ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjFeatures

### LINKED ISSUE #525: md to the main readme
Add mention detection to the main readme: 
https://github.com/CogComp/cogcomp-nlp/blob/master/README.md

### LINKED ISSUE #528: md annotator is not lazy
upon creating a pipeline server, md starts downloading models. 

```
Loading parser from serialized file edu/stanford/nlp/models/lexparser/englishPCFG.ser.gz ... done [1.9 sec].
Downloading the folder from datastore . . .
                GroupId: readonly.org.cogcomp.mention
                ArtifactId: 1.0/ACE_EXTENT.zip
The target /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.mention/1.0/ACE_EXTENT already exists. Skipping download from the datastore . . .
Downloading the folder from datastore . . .
                GroupId: readonly.org.cogcomp.mention
                ArtifactId: 1.0/ACE_HEAD_NONTYPE.zip
The target /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.mention/1.0/ACE_HEAD_NONTYPE already exists. Skipping download from the datastore . . .
Model file read from /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.mention/1.0/ACE_HEAD_NONTYPE/ACE_HEAD_NONTYPE/ACE_NAM.lc
Model file read from /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.mention/1.0/ACE_HEAD_NONTYPE/ACE_HEAD_NONTYPE/ACE_NOM.lc
Model file read from /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.mention/1.0/ACE_HEAD_NONTYPE/ACE_HEAD_NONTYPE/ACE_PRO.lc
Model file read from /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.mention/1.0/ACE_EXTENT/ACE_EXTENT/EXTENT_ACE.lc
Downloading the folder from datastore . . .
                GroupId: readonly.org.cogcomp.gazetteers
                ArtifactId: 1.3/gazetteers.zip
The target /home/khashab2/.cogcomp-datastore/readonly.org.cogcomp.gazetteers/1.3/gazetteers
```

however the correct behavior is to load models only when asked for annotating with mentions.

### PR DESCRIPTION
**Pull request by Slash0BZ**
Closes #525 
Closes #528 
Closes #485

### CODE REVIEW

**Comment by Slash0BZ** [comment]
I will keep commits for other fixes in this PR.

**Comment by danyaljj** [comment]
Also add "closes" statement to the description to close the corresponding issues, upon merging it.

**Comment by mssammon** [comment]
@Slash0BZ if you are not ready for a PR to be reviewed/merged, please add the prefix "WIP" to the title.

File: md/README.md [hunk 1]
```diff
@@ -28,8 +28,24 @@ Extent boundary accuracy given Head
 
 ACE: 89.45
 
+ERE: 88.74
+
+Combined: 86.65
```
Comment (danyaljj) [inline md/README.md]: Do you want to make this a table?

**Comment by danyaljj** [comment]
One minor comment, otherwise looks good to merge.

**Pull request merged**